akka-remote/src/main/scala/akka/remote/MessageSerializer.scala

```diff
@@ -17,6 +18,7 @@ import scala.util.control.NonFatal
  *
  * MessageSerializer is a helper for serializing and deserialize messages
  */
+@InternalApi
```

Comment:
Ah it needs `@InternalApi` because `private[akka]` is not visible from Java, right? Good reminder.

Reply:
Yup!